Control/public/environment/environmentsPage.js

```diff
@@ -61,10 +68,72 @@ export const content = (model) => h('.scroll-y.absolute-fill.text-center', [
  * @param {Array.<Environment>} list
  * @return {vnode}
  */
-const showContent = (model, list) => (list && Object.keys(list).length > 0)
-  ? h('.scroll-auto', environmentsTable(model, list))
-  : h('h3.m4', ['No environments found.']);
+const showContent = (model, list) =>
+  (list && Object.keys(list).length > 0)
+    ? h('.scroll-auto', environmentsTable(model, list))
+    : h('h3.m4', ['No environments found.']);
+
+/**
+ * Show list of create env request to AliECS core as a table
+ * @param {Object} model
+ * @param {array} requests List of requested stored in the backend
+ * @param {Date} now Current date according to backend
+ */
+const showRequests = (model, requests, now) =>
+  (requests && requests.length > 0)
+    ? requestsTable(model, requests, now)
+    : h('h3.m4', ['No requests found.']);
```

Comment:
When looking at the environments page and always raeading "No requests found.", it is a bit confusing as a user.

I would suggest we do not show anything if there is no creation happening.